src/App.jsx

```diff
@@ -1,40 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  const handleTabClick = (tab) => {
+    if (selectedTabId !== tab.id) {
+      setSelectedTabId(tab.id);
+    }
+  };
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+  const isSelected = tabs.find(tab => tab.id === selectedTabId || tab[0]);
```

Comment:
isSelected - means that this variable has boolean type. Consider use correct name.
You chech incorrect
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId)  || tab[0];
```